docs/intro/build-your-first-app.md

```diff
@@ -0,0 +1,95 @@
+
+## 3. Customize Style {#customize-style}
+
+The next step is to customize the style of UI elements. This includes changing the colors, fonts, and sizes of your buttons and labels. In FlutterFlow, you can do this via the [properties panel](#) which provides a range of options for customization.
+
+To customize the style:
+
+1. Adjust the height and color of the Container, and add a border radius.
+2. Change the font weight, size of the quantity label and its value.
+3. Change the color of the increase and decrease buttons to match your design.
+
+
+<iframe src="https://demo.arcade.software/MGpg8TSzMGBusCGyOk89?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="600"></iframe>
+
+## 4. Add State {#add-state}
+
+Once your UI is set up, it's time to make your app interactive by adding a state. This means setting up your app to respond to user interactions. For example, when a user clicks the button to increase the quantity, the number displayed on the label should increase accordingly.
+
+We can achieve this behavior by adding the state variables. A state variable that represents a piece of data that can change over time. For this specific use case, Let’s add a [page state variable](https://www.notion.so/Build-your-first-app-20528f5391f94848aee85e01f2959cd1?pvs=21) that will hold the current quantity value. When a user interacts with the buttons, we update this variable, which in turn updates the UI.
+
+Here's how to add and use state variables:
+
+1. Add a page state variable named ‘productQuantity’ with the *Type* set to Integer and an *Initial Field Value* of 1.
+2. Bind this newly created variable to the Text widget that displays the current product quantity.
+3. When buttons are tapped, add actions to [update the value of the page state variable](#).
+
+
+<iframe src="https://demo.arcade.software/UI92tJF6DX0lOVuidaSH?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="600"></iframe>
+
+## 5. Run the App {#run-app}
+
+Now that you've built and customized your app, it's time to run it. FlutterFlow allows you to test a fully-functional version of your app in [Test](#) and [Run](https://www.notion.so/Build-your-first-app-20528f5391f94848aee85e01f2959cd1?pvs=21) mode. The Run mode typically requires around 2-4 minutes (or more, depending on the size of your project). However, to see your changes immediately, you can run your app in a Test mode that uses Flutter's "[Hot Reload](https://docs.flutter.dev/development/tools/hot-reload)" feature.
```

Comment:
"The Run mode typically requires around 2-4..." Contains excessive qualifiers("typically", "around"). Change to "Run mode requires 2-4..." The range provides enough qualification.